targets/TMS/main.cpp

```diff
@@ -117,7 +119,13 @@ int main() {
     log::LOGGER.log(log::Logger::LogLevel::DEBUG, "Logger initialized.");
     timer.stopTimer();
 
+    // Set up TMS and necessary device drivers
     TMS::TMS tms;
+    IO::PWM& pwm = IO::getPWM<IO::Pin::PC_0>();
+    TMS::HeatPump pump = TMS::HeatPump(pwm);
+    TMS::RadiatorFan fans[] = {
+        TMS::RadiatorFan(IO::getGPIO<IO::Pin::PC_1>()),
+        TMS::RadiatorFan(IO::getGPIO<IO::Pin::PB_0>())};
```

Comment:
I believe the curly brace should be on a new line

Reply:
I think that is correct, the code formatter would have caught that if it didn't match out style